Management into the main branch

Enonome opened the pull request.

Enonome committed:
Update managementArtifacts.md

Error in the postMortem redirection

Enonome committed:
Create postMortem.md

Create the beginning of the Post Mortem Document, it will be finished by Friday of this week (the 25th)

Enonome committed:
Update postMortem.md

Finished the Post Mortem. It is ready to be pushed in the branch and it would be the last commit of this project

EnoGame29 commented on Documents/Management/ManagementArtifacts/postMortem.md:
```diff
@@ -0,0 +1,49 @@
+
+## Project Description
+
+The goal of the project is to create a website that allows users to create and visualize FPGA circuits. The website should be able to parse both .v and .sdf files to create the circuit and display it in a user-friendly way. It will helps students to understand how the FPGA works through the teacher explanations and the website.
+
```
Corrected

EnoGame29 commented on Documents/Management/ManagementArtifacts/postMortem.md:
```diff
@@ -0,0 +1,49 @@
+
+### Management
+
+The management in its entirety was well (4/5). The members found that the things that I set up (such as the GitHub and Slack channels) for the management of the project were very useful. Moreover, they thought that there were a good delegation of the tasks with a good balance of pressure on each member. However, some members felt that I was too stressed and that it showed. <br>
```
Also corrected

EnoGame29 commented on Documents/Management/ManagementArtifacts/postMortem.md:
```diff
@@ -0,0 +1,49 @@
+
+## General Feedback
+
+This project has a lack of organization at the beginning and some members were not very confident about the subject and how to use the framework of the project. Hopefully, over the weeks, the organization improved and the subject (thanks to some questions ask to the client) was better understood creating and guaranteeing a nice atmosphere for all the project.
```
I changed the sentence to be more clear

Enonome committed:
Update postMortem.md

Last management commit, changes have been made to the documents following Copilot's review
